Makefile

```diff
@@ -1,12 +1,17 @@
-PACKAGES := $(shell glide novendor)
+PACKAGES := $(filter-out ., $(shell glide novendor))
 
 export GO15VENDOREXPERIMENT=1
 
+.DEFAULT_GOAL:=build
+
 
 .PHONY: build
 build:
 	go build -i $(PACKAGES)
+	go build -i .
 
+build_prod:
+	env GOOS=linux GOARCH=amd64 $(MAKE)
```

Comment:
Instead of this, I have a temporary branch (just pushed to https://github.com/yarpc/yab/compare/cross_builds?expand=1) that adds a script that will build the binary for a bunch of platforms. I typically use that to build release binaries to upload to the Github release page.

What do you think of that approach? Would prefer not to build a binary called `yab` that is linux/amd64 specific from the makefile directly, since "build_prod" could mean "build with optimizations for production" rather than the architecture.